travis.sh

```diff
@@ -3,11 +3,18 @@ set -e #break script on non-zero exitcode from any command
 gem install bundler
 bundle install
 
+case $OSTYPE in darwin*)
+    brew update > /dev/null
+    brew upgrade cmake
```

Comment:
Is upgrading of CMake really necessary? AFAIK all features used in all contained `CMakeLists.txt/*.cmake` exist in 2.8.12. This includes the Ninja generator: https://cmake.org/cmake/help/v2.8.12/cmake.html#gen:Ninja

I'm asking because generally you want to test with the oldest version of software that you support. This upgrade throws that testing away.